client_src/js/printerManager/templates/gcode-scripts-table.template.js

```diff
@@ -0,0 +1,57 @@
+export function customGcodeScriptTemplate(scripts, scriptLines) {
```

Comment:
Love this idea!

Reply:
Yeah do like this myself it's so much easier and re-usable across the UI